habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/util/DockerfileUtil.java

```diff
@@ -0,0 +1,104 @@
+package org.technologybrewery.habushu.util;
+
+import org.technologybrewery.habushu.HabushuException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class DockerfileUtil {
```

Comment:
A: `DockerfileUtil` sounds a bit too generic given how heavily this class is tied to the functionality of the Mojo.  Something like `ContainerizeDepsDockerfileHelper` might be more clear.